tests/helpers.js

```diff
@@ -134,7 +134,8 @@ export function makeSureItFails(): Function {
  * assert.equal(fakeProcess.exit.called, true);
  *
  */
-export function fake(original: Object, methods: Object = {}): Object {
+ // $FLOW_IGNORE: fake can return any kind of object and fake a defined set of methods for testing.
+export function fake<T>(original: Object, methods: Object = {}): T {
```

Comment:
The above "flow ignore" comment is needed to stop flow from complaining about type errors in the body of the `fake` function itself, but it doesn't seem to affect the checks of the polymorphic type argument, which is what convince flow that `fake` is actually what we use in tests to create any type we want to "fake".